[ADD] fieldservice_stock_location_builder

osi-scampbell opened the pull request:
This PR overrides the create_sub_locations() method in FSMLocationBuilderWizard so all sublocations have the same inventory location as the parent location.

Depends on #326

#129

osi-scampbell committed:
[ADD] fsm_stock_location_builder

[IMP] Add Tests

[IMP] Flake8

[IMP] Remove Dependency

patrickrwilson reviewed (changes requested):
Works well, will approve once all green.

layadelcarmen commented:
I have made a functional review of this module. The wizard creates the new locations with the parent inventory location. Wizard lines needs some validation, admit empty level name and odoo throw an error .`    name + spacer + str(lev_id),
TypeError: unsupported operand type(s) for +: 'bool' and 'str'`

max3903 commented on fieldservice_stock_location_builder/wizard/fieldservice_location_builder_wizard.py:
```diff
@@ -0,0 +1,33 @@
+                new_location = self.env['fsm.location'].create(vals)
+                if num < levels:
+                    build_location(new_location, num + 1)
+        build_location(location, 0)
```
@osi-scampbell I would add a method `prepare_fsm_location_values()` in `fieldservice_location_builder` so you can extend it here just by adding the inventory location to the dictionary.

max3903 commented:
Ping @osi-scampbell

max3903 committed:
[FIX] fieldservice_route_vehicle

max3903 committed:
[FIX] fieldservice_route_vehicle

max3903 committed:
[IMP] fieldservice_route_vehicle

# Conflicts:
#	fieldservice_route_vehicle/models/fsm_route_dayroute.py

OCA-git-bot committed:
fieldservice_account 12.0.3.0.2

OCA-git-bot committed:
[UPD] addons table in README.md

max3903 committed:
[FIX] fieldservice_route

OCA-git-bot committed:
Merge PR #469 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_sale.pot

OCA-git-bot committed:
fieldservice_sale 12.0.2.2.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_sale/

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #470 into 12.0

Signed-off-by max3903

max3903 committed:
[FIX] fieldservice_route

OCA-git-bot committed:
fieldservice_sale_stock 12.0.1.0.2

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #471 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_vehicle.pot

OCA-git-bot committed:
fieldservice_vehicle 12.0.2.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_vehicle
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_vehicle/

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #466 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_route.pot

OCA-git-bot committed:
fieldservice_route 12.0.1.2.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #462 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_vehicle_stock.pot

OCA-git-bot committed:
fieldservice_vehicle_stock 12.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_vehicle_stock
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_vehicle_stock/

OCA-git-bot committed:
[UPD] addons table in README.md

max3903 committed:
[FIX] fieldservice_route_vehicle

OCA-git-bot committed:
Merge PR #467 into 12.0

Signed-off-by max3903

OCA-git-bot committed:
fieldservice_route_vehicle 12.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 88.0% (22 of 25 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_skill
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_skill/es_CL/

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 11.1% (2 of 18 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_account_analytic
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_account_analytic/es_CL/

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 99.1% (534 of 539 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice/es_CL/

osi-scampbell committed:
[IMP] Add Tests

max3903 commented on fieldservice_stock_location_builder/__manifest__.py:
```diff
@@ -10,7 +10,8 @@
         'fieldservice_stock',
-        'fieldservice_location_builder'
+        'fieldservice_location_builder',
+        'fieldservice_account'
```
@osi-scampbell Why do you need `fieldservice_account`?

osi-scampbell replied:
That is where we moved customer_id to

max3903 replied:
@osi-scampbell The `customer_id` field on `fsm_location` is added in `fieldservice_account_analytic`.

1. Let's remove the dependency here
2. In `fieldservice_account_analytic`, add a default value on the field with a method:
```
if self.parent_id:
    return self.parent_id.customer_id.id
return self.owner_id.id
```

osi-scampbell committed:
[IMP] Flake8

osi-scampbell committed:
[IMP] Remove Dependency

osi-scampbell committed:
[IMP] Set Default Customer

[IMP] Move Method Up

[IMP] api.model + space

[IMP] fsm_parent_id

[FIX] Typo

osi-scampbell commented:
#474

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

max3903 committed:
[IMP] Dummy commit to trigger the Weblate sync

osi-scampbell committed:
[IMP] FSO's on VB's

max3903 committed:
[ADD] Spanish (es) translation

OCA-git-bot committed:
Merge PR #476 into 12.0

Signed-off-by max3903

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (5 of 5 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_account_payment
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_account_payment/es_CL/

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 96.3% (26 of 27 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_activity
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_activity/es_CL/

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 98.3% (57 of 58 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_route
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_route/es_CL/

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (3 of 3 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_route_vehicle
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_route_vehicle/es_CL/

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (5 of 5 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_route_stock
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_route_stock/es_CL/

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 94.4% (17 of 18 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_route_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_route_account/es_CL/

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (8 of 8 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_account/es_CL/

Commit:
Added translation using Weblate (Spanish)

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (12 of 12 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_project
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_project/es_CL/

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (10 of 10 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_crm
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_crm/es_CL/

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (17 of 17 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_google_map
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_google_map/es_CL/

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (2 of 2 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_google_marker_icon_picker
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_google_marker_icon_picker/es_CL/

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 16.7% (6 of 36 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_isp_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_isp_account/es_CL/

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (8 of 8 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_maintenance
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_maintenance/es_CL/

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (3 of 3 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_purchase
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_purchase/es_CL/

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (3 of 3 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_route_vehicle
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_route_vehicle/es/

brian10048 committed:
[FIX] fieldservice_sale

osi-scampbell committed:
[IMP] Filter Location/Contact

[IMP] Remove Unused Import

[IMP] Github Comments

[IMP] Add FSM to Variable

max3903 committed:
[FIX] fieldservice_account: Migration script

OCA-git-bot committed:
Merge PR #477 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_account.pot

max3903 committed:
[FIX] fieldservice_account

max3903 committed:
[FIX] fieldservice_account

OCA-git-bot committed:
fieldservice_account 12.0.3.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_account/

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #474 into 12.0

Signed-off-by max3903

OCA-git-bot committed:
fieldservice_account_analytic 12.0.3.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #455 into 12.0

Signed-off-by max3903

OCA-git-bot committed:
fieldservice_isp_account 12.0.2.1.1

OCA-git-bot committed:
[UPD] addons table in README.md

brian10048 committed:
[IMP] fieldservice: Fix team stages

Only show stages for the appropriate team on order kanban and form views

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (18 of 18 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_account_analytic
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_account_analytic/es/

OCA-git-bot committed:
Merge PR #480 into 12.0

Signed-off-by max3903

brian10048 committed:
[IMP] fieldservice: Tooltip for kanban stages

max3903 committed:
[FIX] fieldservice_route: Issue with locale

max3903 committed:
[FIX] fieldservice_route

Commit:
Update test_fsm_order.py

OCA-git-bot committed:
Merge PR #478 into 12.0

Signed-off-by brian10048

oca-travis committed:
[UPD] Update fieldservice_sale.pot

OCA-git-bot committed:
fieldservice_sale 12.0.2.2.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_sale/

OCA-git-bot committed:
[UPD] addons table in README.md

max3903 committed:
[ADD] fieldservice_route: Migration script

max3903 committed:
Merge remote-tracking branch 'ursais/12.0-fix-fieldservice_route' into 12.0-fix-fieldservice_route

max3903 committed:
[IMP] fieldservice_stock: Add warehouse to territory

dreispt committed:
[FIX] Don't error if some moves were deleted by previous logic

When partially receiving Products that require Lot numbers,
creating the backorder would raise an error, caused by missing records.
Invalidating cache or rebrowsing the records did not fix the issue.

Closes #492

osi-scampbell committed:
[DEL] Remove fsm_location_person.py

OCA-git-bot committed:
Merge PR #494 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_account_analytic.pot

OCA-git-bot committed:
fieldservice_account_analytic 12.0.3.2.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_account_analytic
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_account_analytic/

OCA-git-bot committed:
[UPD] addons table in README.md

max3903 committed:
[IMP] fieldservice_route

max3903 committed:
[FIX] fieldservice_vehicle_stock

max3903 committed:
[FIX] fieldservice_route

Commit:
Translated using Weblate (Spanish)

Currently translated at 90.0% (18 of 20 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_vehicle_stock
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_vehicle_stock/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (43 of 43 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_stock
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_stock/es/

OCA-git-bot committed:
Merge PR #481 into 12.0

Signed-off-by brian10048

oca-travis committed:
[UPD] Update fieldservice.pot

OCA-git-bot committed:
fieldservice 12.0.2.3.2

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice/

max3903 committed:
[IMP] fieldservice_stock: Views and doc

OCA-git-bot committed:
Merge PR #493 into 12.0

Signed-off-by max3903

OCA-git-bot committed:
fieldservice_stock 12.0.1.2.1

OCA-git-bot committed:
[UPD] addons table in README.md

max3903 committed:
[IMP] fieldservice_sale_stock

max3903 committed:
[IMP] fieldservice_vehicle_stock

eduaparicio committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 71.2% (384 of 539 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_repair
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_repair/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (8 of 8 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_maintenance
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_maintenance/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (20 of 20 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_vehicle
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_vehicle/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1 of 1 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_sale_stock
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_sale_stock/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 50.0% (1 of 2 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_google_marker_icon_picker
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_google_marker_icon_picker/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3 of 3 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_purchase
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_purchase/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_stock_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_stock_account/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_stock_account_analytic
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_stock_account_analytic/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 20.0% (1 of 5 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_stage_server_action
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_stage_server_action/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 73.5% (396 of 539 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice/pt_BR/

max3903 committed:
[FIX] fieldservice_stock

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 95.0% (512 of 539 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (15 of 15 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_isp_flow
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_isp_flow/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (15 of 15 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_agreement
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_agreement/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (17 of 17 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_account_analytic
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_account_analytic/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (9 of 9 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_account/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (10 of 10 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_crm
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_crm/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_distribution
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_distribution/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (12 of 12 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_project
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_project/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 75.6% (31 of 41 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_sale/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 84.2% (16 of 19 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_fleet
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_fleet/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 86.1% (118 of 137 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_recurring
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_recurring/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 76.2% (16 of 21 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_sale_recurring
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_sale_recurring/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (25 of 25 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_skill
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_skill/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_delivery
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_delivery/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 93.8% (15 of 16 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_substatus
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_substatus/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 55.6% (5 of 9 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_geoengine
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_geoengine/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (22 of 22 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_location_builder
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_location_builder/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 94.1% (16 of 17 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_google_map
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_google_map/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 73.2% (41 of 56 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_partner_multi_relation
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_partner_multi_relation/pt_BR/

patrickrwilson committed:
[IMP] Add security groups

Add security groups for invoices smart button on FSO and FSO smart button on vendor bill. Also added security group to vendor bill list view.

OCA-git-bot committed:
Merge PR #489 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_stock.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
fieldservice_stock 12.0.1.3.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_stock
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_stock/

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #495 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_vehicle_stock.pot

OCA-git-bot committed:
fieldservice_vehicle_stock 12.0.1.1.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_vehicle_stock
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_vehicle_stock/

max3903 committed:
[FIX] fieldservice_route

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #498 into 12.0

Signed-off-by max3903

OCA-git-bot committed:
fieldservice_sale_stock 12.0.1.0.3

OCA-git-bot committed:
[UPD] addons table in README.md

max3903 committed:
[FIX] fieldservice_route: Comments

OCA-git-bot committed:
Merge PR #482 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_route.pot

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_route
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_route/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (65 of 65 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_route
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_route/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 83.3% (15 of 18 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_route_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_route_account/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_google_marker_icon_picker
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_google_marker_icon_picker/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 75.0% (27 of 36 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice_isp_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_isp_account/pt_BR/

marcelsavegnago committed:
[FIX] Correcao de dependencias

murtuzasaleh committed:
[12.0][IMP] Improved Color code functionality.

murtuzasaleh committed:
[12.0][IMP] Added Choose icon color picker field

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 95.0% (512 of 539 strings)

Translation: field-service-12.0/field-service-12.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice/pt_BR/

OCA-git-bot committed:
Merge PR #501 into 12.0

Signed-off-by max3903

OCA-git-bot committed:
fieldservice_vehicle_stock 12.0.1.2.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #500 into 12.0

Signed-off-by max3903

OCA-git-bot committed:
fieldservice_account 12.0.3.1.1

OCA-git-bot committed:
[UPD] addons table in README.md

dreispt committed:
[FIX] Don't error when computing default values

Closos #505

OCA-git-bot committed:
Merge PR #479 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice_account_analytic.pot

OCA-git-bot committed:
fieldservice_account_analytic 12.0.3.3.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_account_analytic
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_account_analytic/

OCA-git-bot committed:
Merge PR #506 into 12.0

Signed-off-by dreispt

OCA-git-bot committed:
fieldservice_route_vehicle 12.0.1.2.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #503 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice.pot

oca-travis committed:
[UPD] Update fieldservice_google_marker_icon_picker.pot

OCA-git-bot committed:
fieldservice_google_marker_icon_picker 12.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_google_marker_icon_picker
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_google_marker_icon_picker/

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #502 into 12.0

Signed-off-by max3903

oca-travis committed:
[UPD] Update fieldservice.pot

oca-travis committed:
[UPD] Update fieldservice_google_map.pot

OCA-git-bot committed:
fieldservice 12.0.2.3.3

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-12.0/field-service-12.0-fieldservice_google_map
Translate-URL: https://translation.odoo-community.org/projects/field-service-12-0/field-service-12-0-fieldservice_google_map/

OCA-git-bot committed:
[UPD] addons table in README.md

osi-scampbell committed:
[IMP] Simplify

osi-scampbell committed:
Merge branch '12.0-add-fsm_stock_location_builder' of https://github.com/ursais/field-service into 12.0-add-fsm_stock_location_builder
